app/controllers/about_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
     @smart_proxies = SmartProxy.authorized(:view_smart_proxies).includes(:features)
     @compute_resources = ComputeResource.authorized(:view_compute_resources)
     @plugins = Foreman::Plugin.all
+    @safemode = ObjectSpace.each_object(Class).select{|x| x < Safemode::Jail }.collect{|x| [x.name.gsub(/::Jail$/, ''), x.allowed_methods.sort.join(' ')]}.sort{|a,b| a.first <=> b.first}
```

Comment:
Please address the Code Climate result, this is very complex. It should be moved into a helper at the very least, and split into multiple lines with meaningful variable names.